database/migrations/1733687303819_create_student_organizations_student_organization_tags_table.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseSchema } from "@adonisjs/lucid/schema";
+
+export default class extends BaseSchema {
+  protected tableName = "student_organizations_student_organization_tags";
```

Comment:
this table name is kinda ehhhhhhhh
but i guess it'll make it easier for adonis to detect it as a pivot table

Reply:
i know it looks weird, but it was my idea to keep the naming convention